app/models/metric/capture.rb

```diff
@@ -13,28 +13,27 @@ def self.capture_cols
   end
 
   def self.historical_days
-    (Settings.performance.history.initial_capture_days || 7).to_i
+    Settings.performance.history.initial_capture_days.to_i
   end
 
   def self.historical_start_time
     historical_days.days.ago.utc.beginning_of_day
   end
 
   def self.concurrent_requests(interval_name)
-    requests = Settings.performance.concurrent_requests[interval_name]
-    requests ||= interval_name == 'realtime' ? 20 : 1
-    requests = 20 if requests < 20 && interval_name == 'realtime'
-    requests
+    ::Settings.performance.concurrent_requests[interval_name]
```

Comment:
Sorry, you are right...I read it backwards.  I meant to say it implements a lower bound.  So, if someone changes it to 1, we push it back up to 20.  This prevents uses from configurating a value that is way too frequent for our system (or the native provider) to handle.